src/Microsoft.Health.Dicom.Core/Exceptions/DicomElementValidationException.cs

```diff
@@ -5,30 +5,34 @@
 
 using Dicom;
 using EnsureThat;
+using Microsoft.Health.Dicom.Core.Features.Validation;
 
 namespace Microsoft.Health.Dicom.Core.Exceptions
 {
     public class DicomElementValidationException : ValidationException
     {
-        public DicomElementValidationException(string name, string value, DicomVR vr, string message)
+        public DicomElementValidationException(ValidationErrorCode errorCode, string name, DicomVR vr, string message, string value = null)
```

Comment:
can we have overloaded constructor, instead of figuring our which param results in which message?